app/javascript/lib/vue-components/modules/Table.vue

```diff
@@ -0,0 +1,240 @@
+                <td
+                  :key="key"
+                  :class="cssClass"
+                  class="gobierto-table__td"
+                >
+                  <span>
+                    {{ item[field] }}
+                  </span>
+                </td>
+              </template>
+              <template v-else-if="type === 'link'">
+                <td
+                  :key="key"
+                  :class="cssClass"
+                  class="gobierto-table__td"
+                >
+                  <span>
+                    <a href="#">{{ item[field] }}</a>
+                  </span>
+                </td>
+              </template>
+              <template v-else-if="type === 'truncate'">
+                <td
+                  :key="key"
+                  class="gobierto-table__td"
+                >
+                  <span :class="cssClass">
+                    {{ item[field] }}
+                  </span>
+                </td>
+              </template>
+              <template v-else>
+                <td
+                  :key="key"
+                  :class="cssClass"
+                  class="gobierto-table__td"
+                >
+                  <span>{{ item[field] }}</span>
+                </td>
+              </template>
+            </template>
+          </router-link>
+        </template>
+      </tbody>
+    </table>
+    <Pagination
+      :data="rowsSorted"
+      :items-per-page="itemsPerPage"
+      :container-pagination="containerPagination"
+      @showData="updateData"
+    />
+  </div>
+</template>
+<script>
+
+import Pagination from "./Pagination.vue";
+import SortIcon from "./SortIcon.vue"
+import { VueFiltersMixin } from "lib/vue/filters"
+export default {
+  name: 'Table',
+  components: {
+    SortIcon,
+    Pagination
+  },
+  mixins: [VueFiltersMixin],
+  defaults: {
+    sortColumn: "id",
+    sortDirection: "up",
+  },
+  props: {
+    data: {
+      type: Array,
+      default: () => []
+    },
+    columns: {
+      type: Array,
+      default: () => []
+    },
+    orderColumn: {
+      type: String,
+      default: ''
+    },
+    showColumns: {
+      type: Array,
+      default: () => []
+    },
+    routingMember: {
+      type: String,
+      default: ''
+    },
+    routingId: {
+      type: String,
+      default: ''
+    },
+    paginationId: {
+      type: String,
+      default: ''
+    },
+    itemsPerPage: {
+      type: Number,
+      default: 0
+    }
+  },
+  data() {
+    return {
+      map: new Map(),
+      currentSortColumn: this.$options.defaults.sortColumn,
+      currentSort: this.$options.defaults.sortDirection,
+      visibleColumns: this.showColumns,
+      filterColumns: [],
+      dataTable: [],
+      containerPagination: this.paginationId
+    };
+  },
+  computed: {
+    tmpRows() {
+      return this.data || []
+    },
+    rowsSorted() {
+      const id = this.currentSortColumn.field || this.orderColumn;
```

Comment:
`this.currentSortColumn` is an String by default. Then it becomes an object? What's that?